src/common/services/fileService.js

```diff
@@ -15,6 +15,11 @@ export const uploadFile = async file => {
     return upload(url, data);
 };
 
+export const uploadLink = async link => {
+    const url = `${REACT_APP_API_ROOT}/files/url?url=${link}`;
```

Comment:
Handle errors from server when URL is invalid or file can't be downloaded via specified IRL: 
![image](https://user-images.githubusercontent.com/31737885/232803161-fa70118c-3085-4212-b6c9-65edd0c6a59c.png)